canonical-web-and-design/ubuntu.com
# Build /managed page

## Done

- Moved the existing `/managed` page to `managed/apps`
- Created `/managed` page
- Created [Managed Services Contact us](https://docs.google.com/document/d/18bgYyreMlSzsMKI5AaTQU1Qjz8eEkVz41Du9q60ZRiY/edit#heading=h.dxn10eelvkoc) modal

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/managed
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check page against [copy doc.](https://docs.google.com/document/d/19GoMudyrd9Zt1e7kfj-koL-Lt0RGYzWHRPuuKYUPvqw/edit#heading=h.mnstvknzgivc)
- Check against [design](https://github.com/canonical-web-and-design/web-squad/issues/4319)


## Issue / Card

Fixes [#4320](https://github.com/canonical-web-and-design/web-squad/issues/4320)

## Screenshots
<img width="1440" alt="Managed IT Services for Openstack, Kubernetes and open source applications | Ubuntu (2021-08-20 09-18-45)" src="https://user-images.githubusercontent.com/57550290/130203207-e017be81-15a1-4d0e-9476-d914ae8bf439.png">
